crates/alerter/src/main.rs

```diff
@@ -25,11 +29,32 @@ struct Order {
     sell_token: H160,
     #[serde(with = "u256_decimal")]
     sell_amount: U256,
+    #[serde(with = "u256_decimal")]
+    fee_amount: U256,
     uid: OrderUid,
     status: OrderStatus,
     creation_date: DateTime<Utc>,
     partially_fillable: bool,
     is_liquidity_order: bool,
+    #[serde(flatten)]
+    class: OrderClass,
+}
+
+impl Order {
+    fn effective_sell_amount(&self) -> U256 {
```

Comment:
This is needed to not false-ly flag limit orders that are in the money but not tradable because of the surplus fee.